frontend/packages/core/src/Feedback/alert.tsx

```diff
@@ -76,10 +77,17 @@ const iconMappings = {
 export interface AlertProps
   extends Pick<MuiAlertProps, "severity" | "action" | "onClose" | "elevation" | "variant"> {
   title?: React.ReactNode;
+  fullWidth?: boolean;
```

Comment:
hmmm I'm curious about this. When looking at the [storybook component](https://storybook.clutch.sh/?path=/story/core-feedback-alert--with-title&args=variant:standard) its rendering as full width of the available space so we might not need this.

Reply:
For whatever reason I couldn't get it to go full width. The alert was fine, but the error rendering the alert was breaking it and I couldn't get it to override. I'll take another pass though.

Reply:
Bah figured it out, was an issue with the grid. I've removed the properties now so should be good :)

Reply:
👍